infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/KubernetesPluginsToolingApplier.java

```diff
@@ -67,6 +72,12 @@ public void apply(InternalEnvironment internalEnvironment, Collection<ChePlugin>
         addSidecar(pod, container, chePlugin, kubernetesEnvironment);
       }
     }
+
+    if (isAuthEnabled) {
+      // enable per-workspace security with JWT proxy for sidecar based WS
+      // because it is the only WS security implementation supported for now
```

Comment:
please, don't use `WS` abbreviation where it is possible since there is a collision with Workspace and WebSocket meaning.

Reply:
fixed